Description: Disable JIT on MIPS*
 Due to QTBUG-58567 we need to disable JIT on mips* in order
 to be able to go forward with Qt on Debian.
Author: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Forwarded: not-needed
Last-Update: 2017-07-31

--- a/src/qml/jsruntime/qv4global_p.h
+++ b/src/qml/jsruntime/qv4global_p.h
@@ -107,7 +107,7 @@
 #  if defined(Q_OS_LINUX)
 #    define V4_ENABLE_JIT
 #  endif
-#elif defined(Q_PROCESSOR_MIPS_32) && defined(Q_OS_LINUX)
+#elif defined(Q_PROCESSOR_MIPS_32) && 0
 #  define V4_ENABLE_JIT
 #endif
 
